libpromises/policy_server.h

```diff
@@ -0,0 +1,72 @@
+/*
+   Copyright (C) CFEngine AS
+
+   This file is part of CFEngine 3 - written and maintained by CFEngine AS.
+
+   This program is free software; you can redistribute it and/or modify it
+   under the terms of the GNU General Public License as published by the
+   Free Software Foundation; version 3.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+  To the extent this program is licensed as part of the Enterprise
+  versions of CFEngine, the applicable Commercial Open Source License
+  (COSL) may apply to this file if you as a licensee so wish it. See
+  included file COSL.txt.
+*/
+
+/** @file
+ * @brief Access to Policy Server IP Address, hostname and port number.
+ *
+ * Provides a simple get/set interface for the policy server variables.
+ * Does hostname resolution behind the scenes. 
+ */
+
+#ifndef CFENGINE_POLICYSERVER_H
+#define CFENGINE_POLICYSERVER_H
+
+#include <cf3.defs.h>
+
+/**
+ * @brief Sets both internal C variables as well as policy sys variables.
+ *
+ * Called at bootstrap and after reading policy_server.dat.
+ * Changes sys.policy_hub, sys.policy_hub_ip and sys.policy_hub_port.
+ *
+ * @param ctx EvalContext is used to set related variables
+ * @param new_policy_server can be 'host:port', same as policy_server.dat
+ */
+void PolicyServerSet(const char *new_policy_server);
```

Comment:
Don't duplicate the doc comments, having them in the implementation file is enough.